spec.md

```diff
@@ -501,14 +503,64 @@ Content-type: application/json
   "type": ["Space"],
   "name": "Example space #1",
   "controller": "did:key:z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW",
-  "linkset": "/space/81246131-69a4-45ab-9bff-9c946b59cf2e/linkset"
+  "linkset": "/space/81246131-69a4-45ab-9bff-9c946b59cf2e/linkset",
+  "totalItems": 0
+}
+```
+
+Example success response (resources have been added to the space):
+
+* Contents listed in a format inspired by [ActivityStreams 2](https://www.w3.org/TR/activitystreams-core/#collections)
+  Collections
+* No pagination used
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "id": "81246131-69a4-45ab-9bff-9c946b59cf2e",
+  "type": ["Space"],
+  "name": "Example space #1",
+  "controller": "did:key:z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW",
+  "linkset": "/space/81246131-69a4-45ab-9bff-9c946b59cf2e/linkset",
+  "totalItems": 2,
+  "items": [
```

Comment:
imho this property should not be on the space response, only on a space collection response '/space/{uuid}/` with trailing slash.